app/src/main/java/org/oppia/app/walkthrough/welcome/WalkthroughWelcomeViewModel.kt

```diff
@@ -0,0 +1,10 @@
+package org.oppia.app.walkthrough.welcome
+
+import androidx.databinding.ObservableField
+import org.oppia.app.viewmodel.ObservableViewModel
+
+/** [ViewModel] for displaying User profile details in walkthrough Welcom Fragment. */
```

Comment:
Nit: `/** [ViewModel] for displaying User profile details in walkthrough Welcome Fragment. */`